### PR TITLE
Adding Czech translation

### PR DESCRIPTION
**Pull request by ZDVokoun**
I'm sending Czech translation of JDM

### CODE REVIEW

**Comment by ZDVokoun** [comment]
Checks are failing because of JGMD data in cz.json

File: _data/trans/cz.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    "extensionp5": "Chrome Web Store",
+    "extensionp6": "",
+    "footercredits": "Vytvořeno",
+    "guide": "Návod",
+    "guideeasy": "Snadný průběh",
+    "guideexplanations": "Odkazy jsou barevně označeny tak, aby naznačovali obtížnost smazání účtu:",
+    "guidehard": "Nemůžou být zcela smazány bez kontaktování podpory",
+    "guideimpossible": "Nemůžou být smazány",
+    "guidemedium": "Potřeba provést několik kroky navíc",
+    "hideinfo": "skrýt podrobnosti...",
```
Comment (tupaschoal) [inline _data/trans/cz.json:24]:
```suggestion
    "hideinfo": "skrýt podrobnosti...",
    "jgmd": "Navštivte Just Get My Data",
```

**Review by tupaschoal — COMMENTED** [review]
I merged the `jgmd` so let me get it back here.

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**